interacting-dom/app.js

```diff
@@ -0,0 +1,52 @@
+const btnCreate = document.getElementById("btn-main");
+const btnToggle = document.querySelector(".btn-toggle");
+const btnRemove = document.querySelector(".btn-remove");
+const taskList = document.querySelector(".list-container ul");
+const remove = document.querySelectorAll('li');
+for (let i = 0; i < remove.length; i++) {
+    attachRemoveButton(remove[i]);
+}
+
+function attachRemoveButton(li) {
+    let remove = document.createElement('button');
+    remove.className = 'remove';
+    remove.textContent = 'Remove';
+    li.appendChild(remove);
+}
+taskList.addEventListener("click", (event) => {
```

Comment:
Should the target for this click event be more specific?
It looks like it is targeting the whole taskList and then checking if the button was selected.
Can you narrow it down to only attach the event to the button itself?